docs/getting-started/quick-start-watt.md

```diff
@@ -118,14 +79,12 @@ npm start
 
 This will internally run `wattpm start` and start your Watt server.
 
-If you want to have have a "watch mode" to automatically restart the server when you make changes, you can run:
+If you want to have a "watch mode" to automatically restart the server when you make changes, you can run:
 
 ```bash
-npm run dev
```

Comment:
This did not work for me. also not added to the package.json of the node app. Shall I remove it?

Reply:
I did that. Also added a note for `npm run start` as this has a different behavior when executed in the root dir compared to the service directory level.